virocon/contours.py

```diff
@@ -72,6 +94,22 @@ def sort_points_to_form_continuous_line(x, y, search_for_optimal_start=False):
 
 
 def save_contour_coordinates(contour, path, model_desc=None):
+    """
+    Saves the coordinates of the calculated contour.
+    
+    Parameters
+    ----------
+    contour : Contour
+    path : string
+     Indictaes the path, where the contour coordinates are saved.
+    model_desc : boolean, optional
+     If true, the algorithm also includes a model description in the header. 
```

Comment:
The model description is a dictionary, not a boolean. 

"The description of the model. model_desc has the keys 'names', 'symbols' and 'units'. Each value is a list of strings. For each dimension of the model the strings describe the name, symbol or unit of that dimension, respectively.
This information is used as the header of the created file.
Defaults to a dict with general descriptions."